benchexec/tablegenerator/react-table/src/tests/__snapshots__/TaskDefinitionViewer.test.js.snap

```diff
@@ -0,0 +1,684 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Render TaskDefinitionViewer for empty_file.yml 1`] = `
+<pre
+  className="link-overlay-text"
+>
+  <span>
+    null
+
+  </span>
+</pre>
+`;
+
+exports[`Render TaskDefinitionViewer for input_files_as_mapping.yml 1`] = `
+<pre
+  className="link-overlay-text"
+>
+  <span>
+    format_version: "1.0"
+
+input_files:
+  - input_file: false_sub_task.yml
+  - input_file: false_other_sub_task.yml
+
+properties:
+  - property_file: 
+  </span>
+  <span
+    className="link-overlay-file-link"
+    onClick={[Function]}
+  >
+    test.prp
+  </span>
+  <span>
+    
+    expected_verdict: false
+    subproperty: other_subproperty
+
+  </span>
+</pre>
+`;
+
+exports[`Render TaskDefinitionViewer for input_files_brackets.yml 1`] = `
+<pre
+  className="link-overlay-text"
+>
+  <span>
+    format_version: "1.0"
+
+input_files:
+  [
```

Comment:
Content like `input_files: [...]` is split across multiple lines, is it possible to avoid this?

Reply:
Hm, this seems to be related to the line length of the list. When adding the splitter tags around the input files, the line exceeds a certain amount which results in the parser splitting it across multiple lines like in your example above.

This occurs every time the amount is exceeded, so even without the splitter tags there will be the same behavior if the names of the input files are too long or there are too many in a single line.

Unfortunately I couldn't find a way to disable this behavior in the lib that is used. I guess there probably won't be a way to avoid this without a very hacky workaround.